src/defaultV2-theme/variables.scss

```diff
@@ -34,7 +34,7 @@ $green: var(--green, #19b394);
 $green-light: var(--green-light, rgba(25, 179, 148, 0.1));
 $blue-light: var(--blue-light, rgba(67, 127, 217, 0.1));
 
-$font-family: "Open Sans";
+$font-family: "Open Sans", "Roboto", sans-serif;
```

Comment:
"Roboto" is non-standard. We need to use: "Helvetica Neue", Helvetica, Arial, sans-serif
After open sans